### PR TITLE
[MM-23483] Add backgroundColor to drawer content parent View

### PR DESCRIPTION
**Pull request by migbot**
#### Summary
The backgroundColor was removed from the drawer content's parent View component, however, this backgroundColor is needed to appropriately display the network indicator banner. iOS is not affected.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-23483

#### Checklist
- [x] Has UI changes

#### Device Information
This PR was tested on:
* Android 10 emulator

#### Screenshots
Screenshot_1585013693

Screenshot_1585013699

### CODE REVIEW

File: app/screens/channel/channel.android.js [hunk 1]
```diff
@@ -72,8 +74,12 @@ export default class ChannelAndroid extends ChannelBase {
...
-const style = StyleSheet.create({
+const getStyle = makeStyleSheetFromTheme((theme) => ({
     flex: {
         flex: 1,
     },
-});
\ No newline at end of file
+    backdrop: {
+        flex: 1,
+        backgroundColor: theme.centerChannelBg,
```
Comment (chuttam) [inline app/screens/channel/channel.android.js]: Does `backgroundColor` introduce redraws in this case? If not, yay, but if so, is there a way of setting `componentBackgroundColor` instead (still using `theme`)?

**Comment by enahum** [comment]
Sure, I can take a look at this

**Comment by enahum** [comment]
@migbot fixed without overdraw, do take a look

**Comment by enahum** [comment]
@chuttam I starting playing with the top position of the NetworkIndicator, saw that it was not transparent, then started playing with the `zIndex` and saw no effect, then I remembered that on Android is about the elevation, and then it was a matter of arranging the components.

**Pull request merged**